Lib/test/test_inspect/test_inspect.py

```diff
@@ -5072,15 +5072,30 @@ def test(a_po, b_po, c_po=3, /, foo=42, *, bar=50, **kwargs):
         self.assertEqual(self.call(test, 1, 2, foo=4, bar=5),
                          (1, 2, 3, 4, 5, {}))
 
-        with self.assertRaisesRegex(TypeError, "but was passed as a keyword"):
-            self.call(test, 1, 2, foo=4, bar=5, c_po=10)
+        self.assertEqual(self.call(test, 1, 2, foo=4, bar=5, c_po=10),
+                         (1, 2, 3, 4, 5, {'c_po': 10}))
 
-        with self.assertRaisesRegex(TypeError, "parameter is positional only"):
-            self.call(test, 1, 2, c_po=4)
+        self.assertEqual(self.call(test, 1, 2, 3, c_po=30, foo=4, bar=5),
```

Comment:
You have not actually fixed this test (and the following one). The value passed to `c_po` (as positional argument) is the same as the default value, so it is not clear whether the argument was used or the default value.

Reply:
Ah, right, sorry. Updated in latest.